src/components/ui/HamburgerMenu.scss

```diff
@@ -1,61 +1,47 @@
 
-
-.navBar {
-  position: relative;
-  overflow: hidden;
+.hamburgerinfotext{
```

Comment:
```suggestion
.hamburgerinfotext {
```

Reply:
Got deleted because of the previous feedback